java/vertx4web/config.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+  website: https://vertx.io/docs/vertx-web/java/
+  version: 4.0.0
+
+build:
+  - mvn package
+
+files:
+  - target/server-jar-with-dependencies.jar
+
+binaries:
+  - target/server-jar-with-dependencies.jar
+
+command: /usr/bin/java -XX:-RelaxAccessControlCheck -XX:+UseNUMA -XX:+UseParallelGC -XX:+AggressiveOpts -jar /opt/web/target/server-jar-with-dependencies.jar
```

Comment:
could you explain those options ? the fact is I'm not a `java` dev trying to review `java` (I think it will be for the better if all jvm options in use, I mean for each frameworks, are the same)

Reply:
They're taken from another java benchmark; unifying them to a property in /.../java seems reasonable but not in the scope of this PR.